.github/issue_template.md

```diff
@@ -0,0 +1,18 @@
+Hi! Thanks for using JupyterHub.
+
+If you are reporting an issue with JupyterHub:
+
+- Please use the GitHub issue search feature to check if your issue has been
+  asked already. If it has, please add your comments to the existing issue.
+
+- Please fill out the details below to help us troubleshoot the issue that
+  you are facing. Please be as thorough as you are able to provide details on
+  the issue.
+
+**How to reproduce the issue**
+
+**What you expected to happen**
+
+**What actually happens**
+
+**Share what version of JupyterHub you are using**
```

Comment:
Can we ask the reporter to run `jupyter troubleshoot` too if he/she is knowledgeable about the command line?